app/src/app/api/v0/inventory/[inventory]/activity-value/route.ts

```diff
@@ -18,10 +20,12 @@ export const POST = apiHandler(async (req, { params, session }) => {
 
   const result = await db.sequelize?.transaction(
```

Comment:
This transaction is getting really big, A good idea would be to break out this logic into a separate ActivityValue Service